bin.js

```diff
@@ -18,9 +18,9 @@ function rulesToString(editorConfigRules) {
 function getEslintRules(filePath) {
   let eslintRules;
   if (filePath !== undefined) {
-    eslintRules = transformer.getEslintRules(path.resolve(filePath));
+    eslintRules = transformer.getEslintRules(path.resolve(process.cwd(), filePath));
```

Comment:
I don't think this does anything unless `filePath` is absolute because [`path.resolve`](https://nodejs.org/api/path.html#path_path_resolve_paths) will incorporate the current working directory.